tests/regression/02-base/71-pthread-once.c

```diff
@@ -0,0 +1,16 @@
+//PARAM: --disable sem.unknown_function.spawn
+#include <pthread.h>
+#include <assert.h>
+
+int g;
+pthread_once_t once = PTHREAD_ONCE_INIT;
+
+void *t_fun(void *arg) {
```

Comment:
I think the correct `init_routine` signature would be:
```suggestion
void t_fun() {
```